app/services/zendesk_drop_off_service.rb

```diff
@@ -82,6 +83,13 @@ def comment_body
 
   private
 
+  def communication_tags
+    [
+      ("email_opt_in" if @drop_off.email.present?),
```

Comment:
Per the tests, we seem to always yield the array ["email_opt_in", "sms_opt_in"].

It would be nice if one of these tests validated our use of the `.present?`.

Not a showstopper.

Reply:
Great! We just pushed some changes that added tests for the individual presence of each tag.

Reply:
:D